executor/src/test/java/com/redhat/service/smartevents/executor/ExecutorServiceTest.java

```diff
@@ -67,16 +82,16 @@ void test(ProcessorDTO processor, String inputEvent, VerificationMode wantedNumb
 
     private static Stream<Arguments> executorServiceTestArgs() {
         Object[][] arguments = {
-                { createSourceProcessor(), BROKEN_JSON, never(), null, null },
-                { createSourceProcessor(), null, times(1), URI.create(ExecutorService.CLOUD_EVENT_SOURCE), "SlackSource" },
-                { createSourceProcessor(), "", times(1), URI.create(ExecutorService.CLOUD_EVENT_SOURCE), "SlackSource" },
-                { createSourceProcessor(), PLAIN_EVENT_JSON, times(1), URI.create(ExecutorService.CLOUD_EVENT_SOURCE), "SlackSource" },
-                { createSinkProcessorWithSameAction(), BROKEN_JSON, never(), null, null },
-                { createSinkProcessorWithSameAction(), PLAIN_EVENT_JSON, never(), null, null },
-                { createSinkProcessorWithSameAction(), createCloudEventString(), times(1), CLOUD_EVENT_SOURCE, CLOUD_EVENT_TYPE },
-                { createSinkProcessorWithResolvedAction(), BROKEN_JSON, never(), null, null },
-                { createSinkProcessorWithResolvedAction(), PLAIN_EVENT_JSON, never(), null, null },
-                { createSinkProcessorWithResolvedAction(), createCloudEventString(), times(1), CLOUD_EVENT_SOURCE, CLOUD_EVENT_TYPE }
+                { createSourceProcessor(), BROKEN_JSON, never(), null, null, false },
+                { createSourceProcessor(), null, times(1), URI.create(ExecutorService.CLOUD_EVENT_SOURCE), "SlackSource", true },
+                { createSourceProcessor(), "", times(1), URI.create(ExecutorService.CLOUD_EVENT_SOURCE), "SlackSource", true },
+                { createSourceProcessor(), PLAIN_EVENT_JSON, times(1), URI.create(ExecutorService.CLOUD_EVENT_SOURCE), "SlackSource", true },
+                { createSinkProcessorWithSameAction(), BROKEN_JSON, never(), null, null, false },
+                { createSinkProcessorWithSameAction(), PLAIN_EVENT_JSON, times(1), "RHOSE", "KafkaTopicSource", true },
```

Comment:
`PLAIN_EVENT_JSON` whilst not a Cloud Event can still be the valid payload of an incoming event.